Documentation/ApiOverview/Backend/JavaScript/Modules/_Modals/_DataModal.html

```diff
@@ -0,0 +1,7 @@
+<a
+    href="delete.php"
+    class="t3js-modal-trigger"
+    data-title="Delete"
+    data-bs-content="Really delete?">
```

Comment:
```suggestion
    data-bs-content="Really delete?"
>
```

Had to look twice where it ends.